src/sentry/monitors/clock_tasks/check_missed.py

```diff
@@ -74,6 +79,7 @@ def dispatch_check_missing(ts: datetime):
             "type": "mark_missing",
             "ts": ts.timestamp(),
             "monitor_environment_id": monitor_environment["id"],
+            "volume_anomaly_result": volume_anomaly_result.value,
```

Comment:
double checking this is in the schema and all good there

Reply:
yeah, mypy and tests would have failed otherwise